AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/scripting_constants.py

```diff
@@ -0,0 +1,34 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Holds constants used across both hydra and non-hydra scripts.
+"""
+
+"""
+Constants for window and tab names
+
+"""
+SCRIPT_CANVAS_UI = "Script Canvas"
+NODE_PALETTE_UI = "Node Palette"
+
+"""
+String constants for Node Palette
```

Comment:
consider in the long run that a dictionary of nodes may be a better accessor than maintaining string constants for each node in the script canvas world.
dictionary key/value would mean that the value can be updated to reflect code changes in the engine while the key is left the same so that test scripts using the key will not require changes to function with the value update.

Reply:
That's a semantical code change TBH. While I like dictionaries for this purpose as you can have highly meaningful and readable coding styles, doing it this way does have the overhead of importing unnessacry code in that dictionary into your tests. So it comes down to "Do we want to improve readability of the test for the marginal loss in test performance?". In this case, I don't foresee the overhead being large enough to see any real notable impact in test performance, but as it is, 

Pros of dictionaries: 

- Better readability (when the dictionaries are grouped accordingly)
     - EG: A dictionary for Node Pallet values, a dictionary for core script canvas editor, a dictionary for QT widgets, etc
- Shorter import lines

Cons of dictionaries:

- More overhead of importing in code that will not be used